packages/sommelier.rb

```diff
@@ -375,17 +412,16 @@ def self.install
 
   def self.postinstall
     # all tasks are done by sommelier.env now
-    now = Time.now.strftime('%Y%m%d%H%M')
-    FileUtils.cp "#{HOME}/.bashrc", "#{HOME}/.bashrc.#{now}"
+    FileUtils.cp "#{HOME}/.bashrc", "#{HOME}/.bashrc.bak"
     system "sed -i '/[sS]ommelier/d' #{HOME}/.bashrc"
-    diff = `diff #{HOME}/.bashrc #{HOME}/.bashrc.#{now}`.chomp
-    if diff == ''
-      FileUtils.rm "#{HOME}/.bashrc.#{now}"
+
+    if FileUtils.identical?("#{HOME}/.bashrc", "#{HOME}/.bashrc.bak")
+      FileUtils.rm "#{HOME}/.bashrc.bak"
```

Comment:
What is wrong with the timestamp?  This could overwrite an existing `.bashrc.bak`.

Reply:
Modified FileUtils. identical logic to use timed backup.